File: third/php/libreria/laws.php
<?php
class Laws{
    private string $name;
    private float $force;
    private string $description;
    public function __construct(string $name, float $force , string $description) {
        $this->name = $name;
        $this->force = $force;
        $this->description = $description;
    }
    public function getName(): string {
        return $this->name;
    }
    public function getForce(): float {
        return $this->force;
    }
    public function getDescription(): string {
        return $this->description;
    }
    public function calculate(): mixed {
        return $this->calculate();
    }
}
?>

File: third/java/Laws.java
public class Laws {
    private String name;
    private float force;
    private String description;

    public Laws (String name, float force, String description) {
        this.name = name;
        this.force = force;
        this.description = description;
    }

    public String getName() {
        return this.name;
    }

    public float getForce() {
        return this.force;
    }

    public String getDescription() {
        return this.description;
    }
    public Object calculate() {
        return "Nada";
    }
}
